.github/workflows/release.yml

```diff
@@ -0,0 +1,105 @@
+#
+# Copyright (c) 2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+name: Upload Release Assets
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  create_release:
+    name: Create release
+    runs-on: ubuntu-latest
+    outputs:
+      upload_url: ${{ steps.create_release.outputs.upload_url }}
+    steps:
+      - name: Create release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: Release ${{ github.ref }}
+          draft: true
+          prerelease: true
+          body: |
+            [MANUALLY ADDED RELEASE NOTES]
+  prepare_environment:
+    name: Prepare Environment
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+      - id: set-matrix
+        run: |
+          echo "::set-output name=matrix::{\"include\":[$( .github/build-container-matrix.sh -m all )]}"
+  build:
+    name: ${{ matrix.productCode }} (${{ matrix.version }}) - Release Asset
+    needs: [prepare_environment, create_release]
+    runs-on: ubuntu-latest
+    strategy:
+      matrix: ${{ fromJson(needs.prepare_environment.outputs.matrix) }}
+    steps:
+      - name: Clone github repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+      - name: Tag name
+        id: tag_name
+        run: echo ::set-output name=sourceTag::${GITHUB_REF#refs/tags/}
+      - name: Docker login
+        uses: azure/docker-login@v1
```

Comment:
I would use the docker action
https://github.com/docker/login-action

Reply:
Changed to `docker/login-action@v1`